frontend/device/android/device.lua

```diff
@@ -375,7 +377,22 @@ function Device:canExecuteScript(file)
     end
 end
 
+function Device:lightDialog()
+    local usleep = require("ffi/util").usleep
+    android.settings.dialog(_("Frontlight settings"), _("Intensity"), _("Warmth"), _("Ok"))
```

Comment:
Nit: not sure about the title: frontlight dialog. On the old widget is called "frontlight", which is true for ereaders. On android "frontlight" only applies to e-ink devices.

Maybe

```lua
local title = android.isEink() and _("Frontlight settings") or _("Light settings")
```
:thinking: Pinging @Frenzie 

Probably brightness is a better name than intensity.

Reply:
If that's about brightness, agreed. :+1:

Reply:
Changed.